### PR TITLE
update the profile creation instruction

### PR DESCRIPTION
**Pull request by MaratMussabekov**
#6466
removed one outdated screenshot since "Create Profile" now a wizard, not a blade

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-cert.md [hunk 1]
```diff
@@ -644,28 +644,28 @@ Sign-in a workstation with access equivalent to a _domain user_.
...
-18. Under **SCEP Server URLs**, type the fully qualified external name of the Azure AD Application proxy you configured. Append to the name **/certsrv/mscep/mscep.dll**. For example, https://ndes-mtephendemo.msappproxy.net/certsrv/mscep/mscep.dll. Click **Add**. Repeat this step for each additional NDES Azure AD Application Proxy you configured to issue Windows Hello for Business certificates. Microsoft Intune round-robin load balances requests among the URLs listed in the SCEP certificate profile.
-19. Click **OK**.
-20. Click **Create**.
+19. Under **SCEP Server URLs**, type the fully qualified external name of the Azure AD Application proxy you configured. Append to the name **/certsrv/mscep/mscep.dll**. For example, https://ndes-mtephendemo.msappproxy.net/certsrv/mscep/mscep.dll. Click **Add**. Repeat this step for each additional NDES Azure AD Application Proxy you configured to issue Windows Hello for Business certificates. Microsoft Intune round-robin load balances requests among the URLs listed in the SCEP certificate profile.
+20. Click **Next**.
+21. Click **Next** two more times to skip **Scope tags** and **Assignments** steps of the wizard and click **Create**.
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-cert.md:668]:
```suggestion
21. Click **Next** two more times to skip the **Scope tags** and **Assignments** steps of the wizard and click **Create**.
```

File: windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-cert.md [hunk 2]
```diff
@@ -644,28 +644,28 @@ Sign-in a workstation with access equivalent to a _domain user_.
...
-   ![WHFB Scep Profile Blade](images/aadjcert/intunewhfbscepprofile-00.png)
-9. The **SCEP Certificate** blade should open. Configure **Certificate validity period** to match your organization.
+5. Select **Windows 10 and later** from the **Platform** list.
+6. Choose **SCEP certificate** from the **Profile** list, and select **Create**.
+7. The **SCEP Certificate** wizard should open. Next to **Name**, type **WHFB Certificate Enrollment**.
+8. Next to **Description**, provide a description meaningful for your environment, then select **Next**.
+9. Select **User** as a certificate type.
+10. Configure **Certificate validity period** to match your organization.
    > [!IMPORTANT]
     > Remember that you need to configure your certificate authority to allow Microsoft Intune to configure certificate validity.
```
Comment (illfated) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-cert.md:654]: possible misalignment issue causing the MD Note blob to break format.
```suggestion
   > Remember that you need to configure your certificate authority to allow Microsoft Intune to configure certificate validity.
```

Comment (mypil) [reply windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-cert.md:654]: @MaratMussabekov - please merge this additional commit. Thanks @illfated

**Comment by mypil** [comment]
@Dansimp @kenwith - Can you please assist to merge this PR? This has been approved by @mapalko 


Thank you.

**Pull request merged**